mk/source.mk

```diff
@@ -226,6 +226,7 @@ endif
 endif
 
 RX_SRC = \
+            $(addprefix drivers/rx/,$(notdir $(wildcard $(SRC_DIR)/drivers/rx/*.c))) \
```

Comment:
Personally, I don't like these automatic includes in large projects.

Reply:
I'm on the fence. I like the idea (under the right structure) that all files in specific locations are automatically included. But it's a bit comme ci comme ça.

Reply:
Sorted.